client/gutenberg/extensions/map-block/info-window/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Wordpress dependencies
+ */
+
+import {
+	Component,
+	createPortal
+} from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+export class InfoWindow extends Component {
+
+	constructor() {
```

Comment:
This method is also removable if we make `closeClick` an arrow function.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/27836/commits/92aa832fd43ffac9ffdbc63ec43c68a9d9346760